Makefile

```diff
@@ -6,7 +6,12 @@ PY_VERSION := 3.11.9
 CUDA_AVAILABLE := $(shell nvidia-smi &> /dev/null; echo $$?)
 PANTS_INSTALLED := $(shell pants --version &> /dev/null; echo $$?)
 PYTHON :=
+#used in docker-compose to choose the right Ray image
+ARCH := 
 
+ifeq ($(UNAME), Darwin)
+	ARCH := -aarch64
```

Comment:
...so this becomes `ARCH := aarch64` 
Also, `UNAME` actually captures the OS. IMHO we should have something like `ARCH := $(shell uname -m)` above, which captures the arch itself (so we don't mix up the arch and the os).

Reply:
Given that:
- different images use different architecture suffixes (e.g. `-aarch64` and `-arm64`) so you cannot directly append arch to an image and expect to find it
- for some images, there is no `-suffix` image for the default architecture (see e.g. [ray](https://hub.docker.com/r/rayproject/ray/tags?page_size=&ordering=&name=latest-py311-cpu))
- we are only interested in capturing the aarch64 suffix for ray right now

... perhaps we might just rename `ARCH` in `RAY_ARCH_PREFIX` so it is semantically accurate, and make it depend on either os being Darwin (`$(UNAME)`) or arch being arm64 (`$(shell uname -m)`)?
I would opt for reading the os as we are not supporting any arm64 platform.